hack/dump/logging-dump.sh

```diff
@@ -0,0 +1,256 @@
+    check_curator_connectivity $pod
+  done
+}
+
+check_kibana_connectivity() {
+  pod=$1
+  echo ---- Connectivity between $pod and elasticsearch >> $kibana_folder/$pod
+  es_host=$(oc get pod $pod  -o jsonpath='{.spec.containers[?(@.name=="kibana")].env[?(@.name=="ES_HOST")].value}')
+  es_port=$(oc get pod $pod  -o jsonpath='{.spec.containers[?(@.name=="kibana")].env[?(@.name=="ES_PORT")].value}')
+  echo "  with ca" >> $kibana_folder/$pod
+  oc exec $pod -c kibana -- curl -ILvs --key /etc/kibana/keys/key --cert /etc/kibana/keys/cert --cacert /etc/kibana/keys/ca -XGET https://$es_host:$es_port &>> $kibana_folder/$pod
+  echo "  without ca" >> $kibana_folder/$pod
+  oc exec $pod -c kibana -- curl -ILkvs --key /etc/kibana/keys/key --cert /etc/kibana/keys/cert -XGET https://$es_host:$es_port &>> $kibana_folder/$pod
+}
+
+check_kibana() {
+  echo -- Checking Kibana health
+  kibana_pods=$(oc get pods | grep -o logging-kibana-[^[:blank:]]*)
+  mkdir $kibana_folder
+  for pod in $kibana_pods
+  do
+    echo ---- Kibana pod: $pod
+    get_env $pod $kibana_folder
+    get_pod_logs $pod $kibana_folder
+    check_kibana_connectivity $pod
+  done
+}
+
+get_elasticsearch_status() {
+  local pod=$1
+  local cluster_folder=$es_folder/cluster-$2
+  mkdir $cluster_folder
+  scurl_es='curl -sv --max-time 5 --key /etc/elasticsearch/secret/admin-key --cert /etc/elasticsearch/secret/admin-cert --cacert /etc/elasticsearch/secret/admin-ca https://localhost:9200'
+  curl_es='curl --max-time 5 --key /etc/elasticsearch/secret/admin-key --cert /etc/elasticsearch/secret/admin-cert --cacert /etc/elasticsearch/secret/admin-ca https://localhost:9200'
+  local cat_items=(health nodes indices aliases)
+  for cat_item in ${cat_items[@]}
+  do
+    oc exec $pod -- $scurl_es/_cat/$cat_item?v &> $cluster_folder/$cat_item
+  done
+  local health=$(oc exec $pod -- $curl_es/_cat/health?h=status)
+  if [ $health != "green" ]
+  then
+    echo Gathering additional cluster information Cluster status is $health
+    cat_items=(recovery shards pending_tasks)
+    for cat_item in ${cat_items[@]}
+    do
+      oc exec $pod -- $scurl_es/_cat/$cat_item?v &> $cluster_folder/$cat_item
+    done
+    oc exec $pod -- $scurl_es/_cat/shards?h=index,shard,prirep,state,unassigned.reason,unassigned.description | grep UNASSIGNED &> $cluster_folder/unassigned_shards
+  fi
+
+}
+
+list_es_storage() {
+  local pod=$1
+  local mountPath=$(oc get pod $pod -o jsonpath='{.spec.containers[0].volumeMounts[?(@.name=="elasticsearch-storage")].mountPath}')
+  echo -- Persistence files -- >> $es_folder/$pod
+  oc exec $pod -- ls -lR $mountPath >> $es_folder/$pod
+}
+
+check_elasticsearch() {
+  echo Checking Elasticsearch health
+  echo -- Checking Elasticsearch health
+  local es_pods=$(oc get pods | grep -o logging-es-[^[:blank:]]*)
+  mkdir $es_folder
+  for pod in $es_pods
+  do
+    echo ---- Elasticsearch pod: $pod
+    get_env $pod $es_folder
+    get_pod_logs $pod $es_folder
```

Comment:
The elasticsearch logs may not be available via `oc logs` if not using `console` logging.  You may need to get 
```
oc exec $pod -- cat /elasticsearch/logging-es/logs/logging-es.log*
```